From f7135e0c8eb4d6860a7bcbfda471bd29c54e591f Mon Sep 17 00:00:00 2001
From: Alexandru Ardelean <alexandru.ardelean@analog.com>
Date: Wed, 30 Oct 2019 09:13:07 +0200
Subject: [PATCH 02/16] [media] axi-hdmi-rx: fix implicit fall-through
 statement

Newer kernels report this warning/error. This change fixes it.

Signed-off-by: Alexandru Ardelean <alexandru.ardelean@analog.com>
(cherry picked from commit 7cee0c95fb5f5c1c06ff1954e1a27a5287ad4f7f)
---
 drivers/media/platform/axi-hdmi-rx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/axi-hdmi-rx.c b/drivers/media/platform/axi-hdmi-rx.c
index 09bc8cf9bc73..7f341479e740 100644
--- a/drivers/media/platform/axi-hdmi-rx.c
+++ b/drivers/media/platform/axi-hdmi-rx.c
@@ -556,6 +556,7 @@ static int axi_hdmi_rx_try_fmt_vid_cap(struct file *file, void *priv_fh,
 		break;
 	default:
 		pix->pixelformat = V4L2_PIX_FMT_RGB24;
+		/* fall-through */
 	case V4L2_PIX_FMT_RGB24:
 	case V4L2_PIX_FMT_BGR24:
 		pix->colorspace = V4L2_COLORSPACE_SRGB;
-- 
2.17.1

